dev-libs/libdbusmenu/libdbusmenu-16.04.0.ebuild

```diff
@@ -0,0 +1,149 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+VALA_MIN_API_VERSION=0.16
+VALA_USE_DEPEND=vapigen
+PYTHON_COMPAT=( python2_7 )
+VIRTUALX_REQUIRED=manual
+
+inherit flag-o-matic multilib-minimal python-single-r1 vala virtualx \
+	xdg-utils
+
+DESCRIPTION="Library to pass menu structure across DBus"
+HOMEPAGE="https://launchpad.net/dbusmenu"
+SRC_URI="https://launchpad.net/${PN/lib}/${PV%.*}/${PV}/+download/${P}.tar.gz"
+
+LICENSE="LGPL-2.1 LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm"
```

Comment:
Because:

```
Keywords for dev-python/dbusmock:
          |                                 |   u   |  
          | a a         p s   a     n r     |   n   |  
          | l m   h i   p p   r m m i i s   | e u s | r
          | p d a p a p c a x m i 6 o s 3   | a s l | e
          | h 6 r p 6 p 6 r 8 6 p 8 s c 9 s | p e o | p
          | a 4 m a 4 c 4 c 6 4 s k 2 v 0 h | i d t | o
----------+---------------------------------+-------+-------
[I]0.16.7 | o ~ ~ o ~ o o o o o o o o o o o | 6 o 0 | gentoo
```

i.e. a happy re-keywording chain.